pymc3/tests/test_distributions.py

```diff
@@ -1354,7 +1352,7 @@ def test_fun(value, mu, sigma):
             Rplus,
             {"mu": Rplus, "sigma": Rplus},
             test_fun,
-            decimal=select_by_precision(float64=5, float32=3),
+            decimal=select_by_precision(float64=3, float32=3),
```

Comment:
We seem to have lost some precision here by using `InverseGamma.get_alpha_beta`,